prac_08/convert_miles_km.py

```diff
@@ -0,0 +1,34 @@
+from kivy.app import App
+from kivy.lang import Builder
+from kivy.properties import StringProperty
+
+M_TO_KM = 1.609344
```

Comment:
M_TO_KM sounds like a dictionary so I would recommend to changed this one to a more appropriate constant